pkg/reconciler/openshift/common/testdata/test-prefix-images-expected.yaml

```diff
@@ -63,7 +63,7 @@ spec:
             "-pr-image", "quay.io/openshift-pipeline/tektoncd-pipeline-pullrequest-init:v0.24.3",
 
             # This is gcr.io/google.com/cloudsdktool/cloud-sdk:302.0.0-slim
-            "-gsutil-image", "gcr.io/google.com/cloudsdktool/cloud-sdk@sha256:27b2c22bf259d9bc1a291e99c63791ba0c27a04d2db0a43241ba0f1f20f4067f",
+            "-gsutil-image", "no-image-available",
```

Comment:
let us keep the `quay.io/openshift-pipeline/tektoncd-pipeline-nop:v0.24.3"` from https://github.com/tektoncd/operator/pull/371/files#diff-18670a9fe9564fddf949b33a4dbcfbbda752c2f740fa0927964b1d516f9a2641R61
as the replacement.

Reply:
@nikhil-thomas just in case if a user is using PipelineResource and if we keep `quay.io/openshift-pipeline/tektoncd-pipeline-nop:v0.24.3` it might give an impression that it not working before bcoz of some error on there side 
they might not know what a `nop` image is for
if we are not publishing an image for that feature so `no-image-available` would make more sense than passing some image that will not work 
wdyt?

Reply:
> it might give an impression that it not working before bcoz of some error on there side

i agree with this 👍  

I would still prefer a pullable image because:
- this image is specific to pipelineResource type storage (specifically gcs). 
   this means we can push the possiblity of the error in to the taskrun pod scope 
- a pullable image will ensure that no other validations at the controller level will break (present now or new ones in the future).
- the enduser will get an `gsutil` executable not found error in the logs. so the user won't be completely clueless.

Reply:
hardcoding the image for `-gsutil-image` is bit tricky as we will not know the tag to use with the image. it will need a separate transformer which will fetch the tag and replace the image.

Reply:
it will be alright, as like other images in this args list, we can specify a toplevel Image replacement as env (already existing feature). The version can be upgraded to the latest during a build/productization.

Reply:
We may also want to see in `tektoncd/pipeline` if there is an issue tracking this and if we can.. have this at pipeline level (aka possibility to not give any image and let pipeline handle that).